test/metabase/test/data/impl.clj

```diff
@@ -98,7 +98,7 @@
     (let [connection-details (tx/dbdef->connection-details driver :db database-definition)
           db                 (db/insert! Database
                                :name    database-name
-                               :engine  (name driver)
+                               :engine  (u/qualified-name driver)
```

Comment:
This fix was because the test tooling didn't work if you tried to create test drivers with namespace-qualified-names. Not actually used in any tests that I ended up keeping around, but still a good fix